### PR TITLE
What I found

### PR DESCRIPTION
**Pull request by KRS-Newberg**

### CODE REVIEW

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 1]
```diff
@@ -67,6 +67,7 @@ I can't forget a single thing about her.
 [sha:168ac9ff9d2764a030af2acf4691b685bae9f016]{
 -- Page 5, Offset 29487.
 -- じっと両手を見つめる。
+//fixedly or fixated - just confirming intent
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:70]: Fixedly, intently

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 2]
```diff
@@ -197,7 +198,8 @@ It's just like it always is.
 [sha:e84500d91ea39c10b3b09c0c125fdfb622e55162]{
 -- Page 13, Offset 29511.
 -- この時間を守るために、危険に身を晒してきた。
-I'd put my life on the line to protect these ordinary moments.
+//'I'd' more imples future rather than past, which is fine, if that is the goal 
+I put my life on the line to protect these ordinary moments.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:202]: Nope, I'd internally written this as a contraction of "I had" not "I would". Past is correct.

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 3]
```diff
@@ -303,6 +305,7 @@ Nothing to do now but stare aimlessly out at the schoolyard.
 [sha:a90140737c9926973a0f5d4701f0fad51a2bc2e0]{
 -- Page 20, Offset 29531. Glued.
 -- 」
+//this is displayed alone on a new line for me
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:308]: Old deepluna / allscr?
2022-05-05-210601_1920x1140_scrot

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 11/11_00_ARC11_1.txt [hunk 4]
```diff
@@ -197,7 +198,8 @@ It's just like it always is.
 [sha:e84500d91ea39c10b3b09c0c125fdfb622e55162]{
 -- Page 13, Offset 29511.
 -- この時間を守るために、危険に身を晒してきた。
-I'd put my life on the line to protect these ordinary moments.
+//'I'd' more imples future rather than past, which is fine, if that is the goal
```
Comment (Avuxo) [inline script/Arcueid/Day 11/11_00_ARC11_1.txt:201]: nope, you got the tense right.

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 11/11_00_ARC11_1.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 11/11_00_ARC11_1.txt]: To be clear I think your change here was good (harder to misread as future)

**Comment by KRS-Newberg** [comment]
Cleaned up comments, implemented all changes

**Comment by KRS-Newberg** [comment]
Cleaned up comments, implemented all changes

**Pull request merged**